doc-site/docs/architecture/distributed_sequencer.md

```diff
@@ -0,0 +1,1437 @@
+ - at that time, it waits until all dispatched transactions are confirmed on chain, then delegates all current inflight transactions to the new coordinator.
+ - if the new coordinator is not up to date with block indexing, then it will reject and the delegation will be retried until it catches up. 
+ - while a node is the current selected coordinator, it sends endorsement requests to every other node for every transaction that it is coordinating
+ -  The endorsement request includes the name of the coordinator node
+ - Each endorsing node runs the selector function to determine if it believes that is the correct coordinator for the current block number
+ - if not, then it rejects the endorsement and includes its view of the current block number in the rejection message
+ - when the coordinator receives the rejection message, it can determine if it is ahead or behind the requested endorser
+ - if the coordinator is ahead, it retries the endorsement request until the endorser catches up and eventually endorses the transaction
+ - if the coordinator is behind, then it waits until its block indexer reaches the next range boundary and delegates all inflight transactions to the new coordinator
+
+
+
+### Sender's responsibility
+
+The sender node for any given transaction remains ultimately responsible for ensuring that transaction is successfully confirmed on chain or finalized as failed if it is not possible to complete the processing for any reason.  While the coordination of assembly and endorsement is delegated to another node, the sender continues to monitor the progress and is responsible for initiating retries or re-delegation to other coordinator nodes as appropriate.
+
+Feedback available to the sender node that can be monitored to track the progress or otherwise of the transaction submission:
+
+ - when the sender node is choosing the coordinator, it may have recently received a heartbeat message from the preferred coordinator or an alternative coordinator
+ - when sending the delegation request to the coordinator, the sender node expects to receive an acknowledgement that the request has been received.  This is not a guarantee that the transaction will be completed.  At this point, the coordinator has only an in-memory record of that delegated transaction
+ - coordinator heartbeat messages.  The payload of these messages contains a list of transaction IDs that the coordinator is actively coordinating
+ - transaction confirmation request.  Once the coordinator has fulfilled the attestation plan, it sends a message to the transaction sender requesting permission to dispatch. If, for any reason, the sender has already re-delegated to another coordinator, then it will reject this request otherwise, it will accept.
```

Comment:
This seems to answer some of my earlier questions, but it also seems inconsistent again. This implies it's the sender that re-delegates (whereas the above diagrams implied some direct old-coordinator <-> new-coordinator comms)